desktop/linux/index.md

```diff
@@ -0,0 +1,157 @@
+---
+description: Docker Desktop for Linux Tech Preview
+keywords: docker, linux, tech preview
+title: Docker Desktop for Linux (Tech Preview)
+toc_min: 1
+toc_max: 2
+---
+
+Welcome to the Docker Desktop for Linux Tech Preview. This Tech Preview is aimed at early adopters who would like to try an experimental build of Docker Desktop for Linux and provide feedback.
+
+Docker Desktop is an easy-to-install application that enables you to build and
+share containerized applications and microservices. Docker Desktop for Linux
+(DD4L) is the second-most popular feature request in our [public
+roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank"
+rel="noopener" class="_"}.
+
+Many developers currently using Docker CE on Linux want feature parity with
+macOS and Windows, with a particular focus on simple installation and update
+management as well as being able to utilize the Docker Desktop GUI.
+Organizations also aspire to provide a consistent, productive, and secure
+development environment across their Mac, Windows, and Linux desktops. Docker
+Desktop for Linux is currently available as a Tech Preview to all developers
+through the free Docker Personal and paid Docker Pro, Team, and Business
+subscriptions.
+
+> **Note**
+>
+> Docker Desktop for Linux is currently offered as a Tech Preview. We recommend
+> that you do not use this in production environments.
+{:.important}
+
+## Download and install
+
+Docker Desktop for Linux is currently available on Ubuntu 21.04, 21.10 and
+Debian distributions.
+
+To install Docker Desktop for Linux:
+
+1. Set up the [Docker repository](../../engine/install/ubuntu.md#install-using-the-repository).
+2. Download and install the Tech Preview Debian package:
+
+  ```console
+  $ curl https://desktop-stage.docker.com/linux/main/amd64/73701/docker-desktop.deb --output docker-desktop.deb
+  $ sudo apt install ./docker-desktop.deb
+  ```
+
+## Check the shared memory
+
+Before you run Docker Desktop for Linux, verify whether the shared memory available on the host is higher than the memory allocated to the VM. By default, Docker Desktop allocates half of the memory and CPU from the host.
+
+```console
+$ df -h /dev/shm
+Filesystem      Size  Used Avail Use% Mounted on
+tmpfs            16G  200M   16G   2% /dev/shm
+```
+
+To set the shared memory size, run:
+
+```console
+$ sudo mount -o remount,size=<the-size-you-want-in-GB> /dev/shm
+```
+
+To ensure this setting persists after a reboot, add the following entry to the `*/etc/fstab*`:
```

Comment:
The directory should  be `/etc/fstab` (without the `*`)

Reply:
Thanks. Updated